.circleci/config.yml

```diff
@@ -28,14 +33,24 @@ jobs:
           name: Verify jq
           command: jq --version
       - run:
-          name: Generate list of chainids and insert into  continue_config.yml file
+          name: Generate Configuration
           command: |
-            sh validation/genesis/validation-inputs/generate-test-config.sh
+            if [ "<< pipeline.parameters.chain >>" = "__required__" ]; then
+              # This will always be selected unless the user chooses to interact with the configuration by setting a non default value in the chain parameter. This would happens if the user launch the pipeline with a specific chain value via either curl or the CircleCI GUI.
+
+              echo "Generating test configuration..."
+              sh validation/genesis/validation-inputs/generate-test-config.sh
+              cp .circleci/continue_config.yml .circleci/selected_continue_config.yml
+            else
+              # This only runs if launched with curl or the CircleCI GUI with non default chain value
+
+              echo "Generating interactive configuration..."
+              cp .circleci/continue_interactive.yml .circleci/selected_continue_config.yml
```

Comment:
Nit: how do you feel about renaming `continue_config.yml` to `main_config.yml` and `continue_interactive.yml` to `artifact_upload_config.yml`? We could keep the `selected_continue_config.yml` filename, I like that.

Reply:
Also, can we rename this job name? Probably outside the scope of the PR but worth doing.

`setup-genesis-allocs-validation` => `generate-and-select-continuation-config`

Reply:
One final thing, can we please update the PR description (for posterity) with a brief description of what changed?

Reply:
regarding the renaming, does generate-test-config.sh do anything with any file? I haven't renamed that for this reason. I can rename if nothing breaks there.

Reply:
Good point, that script would need to be updated. We can do this in a follow up PR if you like, and can test that rename in isolation.